inc/saitypes.h

```diff
@@ -239,6 +239,12 @@ typedef enum _sai_object_type_t
     SAI_OBJECT_TYPE_TAM_HISTOGRAM            = 68,
     SAI_OBJECT_TYPE_TAM_MICROBURST           = 69,
     SAI_OBJECT_TYPE_MAX                      = 70,
+    /* Start of experimental object types */
+    SAI_OBJECT_TYPE_DTEL_QUEUE_REPORT        = SAI_OBJECT_TYPE_MAX,
+    SAI_OBJECT_TYPE_DTEL_INT_SESSION         = SAI_OBJECT_TYPE_MAX + 1,
+    SAI_OBJECT_TYPE_DTEL_REPORT_SESSION      = SAI_OBJECT_TYPE_MAX + 2,
+    SAI_OBJECT_TYPE_DTEL_EVENT               = SAI_OBJECT_TYPE_MAX + 3,
+    SAI_OBJECT_TYPE_EXPERIMENTAL_MAX         = SAI_OBJECT_TYPE_MAX + 4,
```

Comment:
Such construct are not supported also you need to have quad API for each object type to create remove set And get